single_cell/utils/inpututils.py

```diff
@@ -241,22 +241,17 @@ def get_fastqs(fastqs_file):
     return fastq_1_filenames, fastq_2_filenames
 
 
-def load_cohort_qc_inputs(yaml_file):
-    yaml_file = load_yaml(yaml_file)
-    cohort = list(yaml_file.keys())[0]
-    data = yaml_file[cohort]
-
+def load_cohort_qc_inputs(yaml):
```

Comment:
yaml is also a module that's imported at the top of this file. 

please rename yaml here to something else